changelog/unreleased/issue-2277

```diff
@@ -0,0 +1,15 @@
+Enhancement: add support for ppc64le
+
+It was added supported for ppc64le as part of Restic. 
+
+Power is the computer architecture used by the fatest 
+supercomputer available today and also used by many 
+big companies which are in they journey to the 
+multi-cloud environment where Kubernetes is a key 
+component. By adding support for Power as part of 
+Restic we enabled Velero to be enabled on Power and 
+consequently, ennabled several users to use a 
+robust backup solution regardless of their prefered 
+computer architecture.
```

Comment:
The grammar of this text needs to be fixed. Also, it doesn't need all that much background about what POWER is about. And what is Velero anyway? In other words, please make the text more relevant for a changeling entry :)